TuneUp/ProfiledNodeViewModel.cs

```diff
@@ -39,7 +39,35 @@ public string ExecutionTime
             }
         }
 
-        internal NodeModel NodeModel { get; set; }
+        private bool wasExecutedOnLastRun;
```

Comment:
maybe we can call this `currentRun`? same in the UI - what do you think?
previous and last seem too similar to me.

Reply:
Agreed. Updated.